src/command/FileCommands.cxx

```diff
@@ -233,3 +239,103 @@ handle_read_comments(Client &client, Request args, Response &r)
 
 	gcc_unreachable();
 }
+
+InputStreamPtr
+find_stream_art(const char *directory, Mutex &mutex, Cond &cond)
+{
+    static constexpr char const * art_names[] = {
+        "cover.png",
+        "cover.jpg",
+        "cover.tiff",
+        "cover.bmp"
+    };
+    
```

Comment:
Line consisting only of trailing spaces (and there are LOTS more of that, my editor is all in red highlighting those)